cpu/cc3200/periph/timer.c

```diff
@@ -0,0 +1,276 @@
+/*
+ * Copyright (C) 2019 Ludwig Maximilian Universität
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup        cpu_cc3200
```

Comment:
This has been defined many times before (and causes a warning)
I think you meant `@ingroup`.